web/apps/labelstudio/src/components/Description/Description.styl

```
@@ -22,3 +22,10 @@
 
     &_large
       margin 40px 0
+
+  &_alert
+   &_warning
+     background-color rgb(255, 251, 230);
+     border 1px solid rgb(255, 229, 143);
+     border-radius 8px
+     padding 10px
```

ident and missing trailing new line
```suggestion
  &_alert
    &_warning
      background-color rgb(255, 251, 230);
      border 1px solid rgb(255, 229, 143);
      border-radius 8px
      padding 10px

```